osu.Framework/Graphics/Audio/WaveformGraph.cs

```diff
@@ -43,7 +43,7 @@ private void load(ShaderManager shaders)
         private float resolution = 1;
 
         /// <summary>
-        /// Gets or sets the amount of <see cref="Framework.Audio.Track.Waveform.Point"/>'s displayed relative to <see cref="WaveformGraph.DrawWidth"/>.
+        /// Gets or sets the amount of <see cref="Framework.Audio.Track.Waveform.Point"/>'s displayed relative to <see cref="Drawable.DrawWidth"/>.
```

Comment:
This kind of change makes the documentation worse in my opinion. Yes, I know, VS will complain that *member name can be simplified*, but this kind of change loses the *original context* when humans are reading this source. Before it was obvious from the explicit type qualification that this referred to `WaveformGraph`'s draw width, but after the change it's some `Drawable` - *what* `Drawable`? I think `WaveformGraph` should be *at least* explicitly `<see cref="" />`'d here if we want to go this direction.

Reply:
> Yes, I know, VS will complain that _member name can be simplified_

No, the compiler will **warn** that _cannot find the reference_, and make the documentation broken. That's the rule of xmldoc, not the same as language rule - you can omit the class qualification when referring properties from base class in code, but you cannot do so in xmldoc.

Actually it's the IDE's responsibility to show references with suitable qualification. The example is adding namespaces you've not imported.

Reply:
I was pretty sure VS did that grey-out-qualification thing in these sorts of cases? Nevertheless, I still think it should be made clear that the cref is referring to *the current instance's* `DrawWidth`, even if by doing something like
```suggestion
        /// Gets or sets the amount of <see cref="Framework.Audio.Track.Waveform.Point"/>'s displayed relative to this graph's <see cref="Drawable.DrawWidth"/>.
```

Reply:
I'm definitely agree with qualification of current instance.

> I was pretty sure VS did that grey-out-qualification thing in these sorts of cases?

In code, not xmldoc. Basically it's for static members qualified with derived class, and `nameof`.

Reply:
Seems xmldoc references supports things like `<see cref="Drawable.DrawWidth">DrawWidth</see>`, to custom displayed text for a reference. This can be an option, because unresolved reference is treated as plain text, which is definitely worse than this.